client/src/components/AddProduct/index.js

```diff
@@ -14,46 +14,52 @@ import {
 } from 'antd';
 import {
   BsUpload,
-  RiPriceTag2Line,
-  MdPhotoSizeSelectLarge,
+  FaMinus,
+  FaPlus,
   MdDescription,
   MdSubtitles,
 } from 'react-icons/all';
 
 import './style.css';
+import FormItem from 'antd/lib/form/FormItem';
 
 const { Option } = Select;
 
 const AddProduct = ({ showForm, hideForm }) => {
   const [loaded, setLoaded] = useState(false);
   const [error, setError] = useState();
   const [paintingImg, setPaintingImg] = useState();
-  const [category, setCategory] = useState();
 
-  const handleCategories = (value) => {
-    setCategory(value);
-  };
+  const onFinish = async ({ props, size, price, ...values }) => {
+    const obj = {};
+    obj[size] = price;
+    let body;
+    if (props.length !== 0) {
```

Comment:
This line throw error can't read property of undefined so you need to check like this
```suggestion
    if (props) {
```